doc/changes.rst

```diff
@@ -246,7 +246,12 @@ Changes to individual commands
   * Dropped ``--pkg`` option. Positional arguments can now be used to specify packages to check closure for.
 
 ``repolist``
-  * Options ``-v`` and ``--verbose`` have been removed. The functionality is replaced by the ``repoinfo`` command, which was already introduced in DNF4.
+  * The ``repolist`` and ``repoinfo`` commands are now subcommands of the ``repo`` command: ``repo list`` and ``repo info``.
+
+    * Original commands still exist as compatibility aliases.
+
+  * Options ``-v`` and ``--verbose`` have been removed. The functionality is replaced by the ``repo info`` command (already in DNF4 as ``repoinfo``).
+  * When no repositories are enabled, empty output is now provided by default instead of displaying "No repositories available".
```

Comment:
I think it should say `When no repositories are available` instead of `enabled` because  the message doesn't show when there are disabled repositories (`dnf repolist --disablerepo=*` produces empty output).

I am sorry about splitting the review, I didn't notice this detail before.

Reply:
Good point, I didn't realize that.